app/views/portal/show.rb

```diff
@@ -582,11 +592,15 @@ def data_section_field_values(section)
       end
     end
 
-    def data_section_field_search_path(val, field)
+    def data_section_field_search_path(val, field, quoted)
       return unless val.is_a?(String)
 
       search_val = val.gsub(/[()\[\]<>]/, '')
-      search_path(q: "#{field}:#{search_val}")
+      if (quoted)
```

Comment:
Don't use parentheses around the condition of an `if`.<br>Don't use parentheses around a variable.